Codeinwp/neve
# Fix revert advance search

### Summary
Reverted all the changes for advanced search components.

### Will affect the visual aspect of the product
NO


### Test instructions
- Please test with this PR and Neve 3.4.1, no errors should appear
- Please test with this PR an [Neve PR](https://github.com/Codeinwp/neve-pro-addon/pull/2176)
- Please test with [Neve PR ](https://github.com/Codeinwp/neve-pro-addon/pull/2176)and Neve pro 2.4.1



Closes Codeinwp/neve-pro-addon#2174.
